i18n/translations/en/manage-form-access.json

```diff
@@ -0,0 +1,30 @@
+{
+  "accessControlError": "You do not have permissions to manage access to this form",
+  "addPeopleToShareAccess": "Add people to share access",
+  "areYouSure": "Are you sure?",
+  "back": "Back",
+  "cancel": "Cancel",
+  "confirmError": "There was an error",
+  "delete": "Delete",
+  "deleteInvitation": "Delete invitation",
+  "downloadForm": "Download form",
+  "emailAlreadyInList": "{{email}} is already in the list",
+  "error": "There has been an error.",
+  "invalidEmail": "{{email}} is an invalid email address",
+  "invitationFailed": "Unable to send invitation to {{email}}",
+  "invitationSent": "Invitation sent",
+  "invite": "Invite",
+  "invited": "Invited",
+  "inviteToForm": "Invite to form",
+  "loading": "Loading...",
+  "manageFormAccess": "Manage form access",
+  "manageOwnership": "Manage ownership",
+  "mustBeAGovernmentAddress": "Must be a government email address",
+  "next": "Next",
+  "optionalMessage": "Optional message",
+  "peopleWithAccess": "People with access",
+  "remove": "Remove",
+  "resend": "Resend",
+  "templateNotFound": "Template not found {{templateId}}",
+  "userAlreadyHasAccess": "{{email}} already has access"
+}
```

Comment:
```suggestion
{
  "accessControlError": "You do not have permission to manage access to responses.",
  "addPeopleToShareAccess": "Share access",
  "areYouSure": "Are you sure?",
  "back": "Back",
  "cancel": "Cancel",
  "confirmError": "Something went wrong",
  "delete": "Delete",
  "deleteInvitation": "Delete invitation",
  "downloadForm": "Download form",
  "emailAlreadyInList": "{{email}} was already added",
  "error": "There was an error.",
  "invalidEmail": "{{email}} is not a valid email address",
  "invitationFailed": "Unable to send invitation to {{email}}",
  "invitationSent": "Invitation sent",
  "invite": "Invite",
  "invited": "Invited",
  "inviteToForm": "Invite to manage responses",
  "loading": "Loading...",
  "manageFormAccess": "Manage access",
  "manageOwnership": "Manage ownership",
  "mustBeAGovernmentAddress": "Add people with their government email address",
  "next": "Continue",
  "optionalMessage": "Message (optional)",
  "peopleWithAccess": "People with access",
  "remove": "Remove",
  "resend": "Re-send",
  "templateNotFound": "Template not found {{templateId}}",
  "userAlreadyHasAccess": "{{email}} already has access"
}
```

Reply:
What does "Template not found" mean? Is that an internal error?

Reply:
It's a potential but unlikely error. Basically before most actions are taken, we do two basic checks - that the user requesting the invite exists, and that the template exists.